react/features/base/lib-jitsi-meet/native/polyfills-browser.js

```diff
@@ -99,6 +99,8 @@ function _visitNode(node, callback) {
 (global => {
     const { DOMParser } = require('xmldom');
 
+    global.DOMParser = DOMParser;
```

Comment:
Can you please add a comment and say that that lib-jitsi-meet requires it if using WebSockets? (see how we document other polyfills). Thanks!

Reply:
I've added comment in 9b64424b29595361f3f7351325a01660ae519489